qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -548,13 +549,11 @@ def specialize(self):
 
 
 class TwoQubitWeylPartialSWAPFlipEquiv(TwoQubitWeylDecomposition):
-    """U ~ Ud(α𝜋/4, α𝜋/4, -α𝜋/4) ~ SWAP**α
-
+    r""":math:`U \sim U_d(\alpha\pi/4, \alpha\pi/4, -\alpha\pi/4) \sim \text{SWAP}^\alpha`
     (a non-equivalent root of SWAP from the TwoQubitWeylPartialSWAPEquiv
-    similar to how x = (±sqrt(x))**2 )
-
+    similar to how ``x = (±sqrt(x))**2``)
```

Comment:
In case you want to math-mode it
```suggestion
    similar to how :math:`x = (\pm \sqrt(x))^2`)
```